servermaster/server.go

```diff
@@ -686,6 +686,7 @@ func (s *Server) runLeaderService(ctx context.Context) (err error) {
 			log.L().Warn("job manager close with error", zap.Error(err))
 		}
 	}()
+	s.leaderInitialized.Store(true)
```

Comment:
I think we can remove the time.Sleep(time.Second) in e2e_dm_test.go now

Reply:
Yep, use require.Eventually() now.